subcommittees/mentoring/README.md

```diff
@@ -1,70 +1,86 @@
 # The Mentoring Subcommittee
 
-## Purpose and Scope
+## Our mission
```

Comment:
If we want to make this change, we should probably make it to all task forces and subcommittees and also update [the template][1].  I don't really care what the wording is, but the current “Purpose and Scope” is from 19c9d5bd (#102).

[1]: https://github.com/swcarpentry/board/tree/45f7e9606e58c85cc9fbf3f534416b39fd74a7a2/subcommittees/template#purpose-and-scope